packages/common/src/store/pages/collection/reducer.ts

```diff
@@ -33,27 +31,13 @@ export const initialState = {
 }
 
 const actionsMap = {
-  [FETCH_COLLECTION](
-    state: CollectionsPageState,
-    _action: FetchCollectionAction
-  ) {
-    return {
-      ...state,
-      status: Status.LOADING,
-      smartCollectionVariant: null
-    }
-  },
   [FETCH_COLLECTION_SUCCEEDED](
     state: CollectionsPageState,
     action: FetchCollectionSucceededAction
   ) {
     return {
       ...state,
-      collectionId: action.collectionId,
-      collectionUid: action.collectionUid,
-      userUid: action.userUid,
-      status: Status.SUCCESS,
-      collectionPermalink: action.collectionPermalink
+      collectionId: action.collectionId
```

Comment:
if this is all we're using it for, can we just get it from the url params where we need it? or is it selected from in sagas and stuff?

Reply:
So this is needed for the lineup to know which collection is we are working with. I will remove all other state in this or though, good call